### PR TITLE
add task tests

### PR DESCRIPTION
**Pull request by FoseFx**
Adds tests for:
 - GetTasksByPatient
 - GetTasksByPatientSortedByStatus
 - GetAssignedTasks
 - DeleteTask
 - UpdateTask
 - AddSubTask
 - RemoveSubTask
 - UpdateSubTask
 - SubTaskToDone
 - SubTaskToDo
 - TaskToToDo
 - TaskToInProgress
 - TaskToDone
 - AssignTaskToUser
 - UnassignTaskFromUser
 - PublishTask
 - UnpublishTask

### CODE REVIEW

**Review by PaulKalho — CHANGES_REQUESTED** [review]
looks good but not sure about the `UpdateSubTask` test.

File: services/task-svc/integration_tests/33.UpdateSubTask.exec [hunk 1]
```diff
@@ -1,22 +1,47 @@
...
+	-d '{"id": "f531ae7e-96cd-4d5d-ab62-1afbe90fdd87"}' \
+	localhost:3001 \
+	proto.services.task_svc.v1.TaskService/GetTask
+
+
 echo "update"
 grpcurl --plaintext -H "X-Organization: 3b25c6f5-4705-4074-9fc6-a50c28eba406" \
 	-H "Authorization: Bearer eyJzdWIiOiIxODE1OTcxMy01ZDRlLTRhZDUtOTRhZC1mYmI2YmIxNDc5ODQiLCJlbWFpbCI6InRlc3RpbmUudGVzdEBoZWxwd2F2ZS5kZSIsIm5hbWUiOiJUZXN0aW5lIFRlc3QiLCJuaWNrbmFtZSI6InRlc3RpbmUudGVzdCIsIm9yZ2FuaXphdGlvbnMiOlsiM2IyNWM2ZjUtNDcwNS00MDc0LTlmYzYtYTUwYzI4ZWJhNDA2Il19" \
-	-d '{"id": "f2e70feb-9137-44d7-bcad-b0e90bd0706a", "name": "Hi c:"}' \
+	-d '{"id": "f531ae7e-96cd-4d5d-ab62-1afbe90fdd87", "name": "Hi c:"}' \
```
Comment (PaulKalho) [inline services/task-svc/integration_tests/33.UpdateSubTask.exec]: Why? For `updateSubTask` the `id` should be still the one of the subtask, not the task's `id`.

 id should be `f2e70feb-9137-44d7-bcad-b0e90bd0706a` again

**Review by PaulKalho — CHANGES_REQUESTED** [review]

File: services/task-svc/integration_tests/33.UpdateSubTask.exec (hunk 1 above)
Comment (FoseFx) [reply services/task-svc/integration_tests/33.UpdateSubTask.exec]: I should stop blindly copy and pasting. Thank you for catching this!

**Comment by FoseFx** [comment]
Blocked by #607

**Review by PaulKalho — APPROVED** [review]

**Pull request merged**